recipes/bohra/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "bohra" %}
+{% set version = "1.0.19" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f99205fbdad0a1822a0eb3ae72f55d76b1fcd9d03cbe60923b9d74849929e76b"
+
+build:
+  noarch: generic
+  number: 0
+  entry_points:
+    - bohra=bohra.bohra:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
```

Comment:
I think everything can be removed from host except pip and python.

Reply:
@bgruening The python dependencies are all required in host, but you were correct re the rest. I have made these changes and pushed them! Thank yo for the suggestion